Makefile

```diff
@@ -175,15 +175,6 @@ prometheus: node-exporter alert-buffer
 	$(MAKE) apply WHAT=projects/prometheus/prometheus.yaml
 .PHONY: prometheus
 
-prometheus-rules:
-	oc create cm prometheus-rules -n openshift-monitoring --from-file=prometheus.rules=projects/prometheus/prometheus.rules.yaml -o yaml --dry-run | oc apply -f -
-	oc set volume sts/prometheus  -n openshift-monitoring -c prometheus --add -t configmap --configmap-name=prometheus-rules -m /etc/prometheus/prometheus.rules --sub-path=..data/prometheus.rules || true
```

Comment:
This is not working in 3.10; we need to find a different way to lay down our config on top of the prometheus configmap.